apps/judicial-system/web/src/components/InfoCard/DefendantInfo/DefendantInfo.tsx

```diff
@@ -37,31 +37,27 @@ interface DefendantInfoProps {
 }
 
 export const getAppealExpirationInfo = (
-  viewDate?: string | null,
+  verdictAppealDeadline?: string | null,
```

Comment:
Use the deadline already calculated by the api rather than calculating again in the client.